compair/static/modules/assignment/assignment-module.js

```diff
@@ -1210,6 +1215,9 @@ module.controller("AssignmentWriteController",
         $scope.loggedInUserId = resolvedData.loggedInUser.id;
         $scope.canManageAssignment = resolvedData.canManageAssignment;
         $scope.availableCriteria = resolvedData.criteria.objects;
+        $scope.groups = resolvedData.groups.objects;
```

Comment:
This causes exception when tried to duplicate an assignment.  There is no `groups` in `resolvedData`

Reply:
Thanks missed it